packages/grafana-data/src/types/dashboard.cue

```diff
@@ -0,0 +1,213 @@
+package grafanaschema
+
+import "github.com/grafana/grafana/cue/scuemata"
+
+Family: scuemata.#Family & {
```

Comment:
This is where the dashboard scuemata family declaration is currently planned to canonically live. But this isn't actually the version that's used for anything at runtime - rather, that's what's in `cue/data/gen.cue` - so changes to this file actually have no effect until we get the process of assembling CUE files under `packages/` into their target locations under `cue/` in place.